process/block/preprocess/rewardTxPreProcessor.go

```diff
@@ -331,7 +331,6 @@ func (rtp *rewardTxPreprocessor) processRewardTransaction(
 		return err
 	}
 
-	//TODO: These lines could be deleted as in this point these values are already set (this action only overwrites them)
```

Comment:
I would say to let this TODO as these lines really could be deleted from all these methods: func (rtp *rewardTxPreprocessor) processRewardTransaction, func (scr *smartContractResults) processSmartContractResult( and func (txs *transactions) processAndRemoveBadTransaction(

Reply:
done.